options.rb

```diff
@@ -0,0 +1,133 @@
+module STUDENTTEACHER
+  def student_option()
+    print 'Age '
+    age = gets.chomp.to_i
+    print 'Name '
+    name = gets.chomp!
+    print 'Has the parent permission [Y/N]'
+    inputpermission = gets.chomp!
+
+    case inputpermission.downcase
+    when 'y'
+      parent = true
+    when 'n'
+      parent = false
+    else
+      puts 'Error selectc the correct option'
+    end
+    @students << Student.new(age, name, ClassRoom.new(''), '', parent_permission: parent)
+    @people << Student.new(age, name, ClassRoom.new(''), '', parent_permission: parent)
+    puts 'Student added successfully'
+  end
+
+  def teacher_option()
+    print 'Age '
+    age = gets.chomp.to_i
+    print 'Name '
+    name = gets.chomp!
+    print 'Especiality '
```

Comment:
**[OPTIONAL]**

- Please, let's improve our spelling to improve readability while interacting with the app as this would foster a clear comprehension for the users thereby improving the user experience. 

> This should be `specialty` or `specialization`